packages/vue/src/components/molecules/SfModal/SfModal.vue

```diff
@@ -1,5 +1,6 @@
 <script src="./SfModal.js"></script>
 <template lang="html" src="./SfModal.html"></template>
 <style lang="scss">
+@import "../../../utilities/transitions/transitions.scss";
```

Comment:
can't import it here, css would be duplicated because it's already imported on `global.scss`

```suggestion
```